docs/plugins.md

```diff
@@ -90,6 +90,28 @@ $ probot run -a APP_ID -P private-key.pem ./index.js
 Listening on http://localhost:3000
 ```
 
+## HTTP Routes
+
+`robot.router` is an [express](http://expressjs.com/) app that will serve HTTP requests.
+
+```js
+const express = require('express');
+
+module.exports = robot => {
+  // Use any express middleware
+  robot.router.use(express.static(__dirname + '/public'));
+
+  //
+  robot.router.get('/hello-world', (req, res) => {
+    res.end('Hello World');
+  });
+};
```

Comment:
Thanks. I wrote these out before I suggested the other implementation. Updated now.